packages/data-context/src/sources/ProjectConfigDataSource.ts

```diff
@@ -32,7 +32,10 @@ export class ProjectConfigDataSource {
       this.ctx.coreData.currentProject.config = Promise.resolve().then(async () => {
         const configFile = await this.ctx.config.getDefaultConfigBasename(projectRoot)
 
-        return this.ctx._apis.projectApi.getConfig(projectRoot, { configFile })
+        return this.ctx._apis.projectApi.getConfig(projectRoot, {
+          configFile,
+          testingType: this.ctx.wizardData.chosenTestingType ?? undefined,
```

Comment:
Eg this makes little sense - `?? undefined`? It's either `null` or `undefined` be default so this doesn't make a ton of sense. How does this even fix the problem 🤔